### PR TITLE
Nomad docker support

### PR DESCRIPTION
**Pull request by jerrypeng**

### CODE REVIEW

File: heron/tools/apiserver/src/java/com/twitter/heron/apiserver/resources/TopologyResource.java [hunk 1]
```diff
@@ -502,7 +502,8 @@ private Config createConfig(Collection<Pair<String, Object>> keyValues,
   }
 
   private boolean isLocalMode() {
-    return "local".equalsIgnoreCase(getCluster()) || "standalone".equalsIgnoreCase(getCluster());
+    return "local".equalsIgnoreCase(getCluster()) || "standalone".equalsIgnoreCase(getCluster())
+        || "nomad".equalsIgnoreCase(getCluster());
```
Comment (kramasamy) [inline heron/tools/apiserver/src/java/com/twitter/heron/apiserver/resources/TopologyResource.java]: @jerrypeng - is this needed for 'nomad' direct deployment?

**Comment by kramasamy** [comment]
@jerrypeng - can you fix the CI?

File: heron/tools/apiserver/src/java/com/twitter/heron/apiserver/resources/TopologyResource.java (hunk 1 above)
Comment (jerrypeng) [reply heron/tools/apiserver/src/java/com/twitter/heron/apiserver/resources/TopologyResource.java]: no this is for API server

Comment (kramasamy) [reply heron/tools/apiserver/src/java/com/twitter/heron/apiserver/resources/TopologyResource.java]: ok got it - let us fix the CI and merge it.

**Pull request merged**